### PR TITLE
Implementation of Change ownr with classic experience

### PR DESCRIPTION
**Pull request by Enrique-C**

### CODE REVIEW

File: src/main/java/salesforce/ui/pages/owner/OwnerEditClassicPage.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    private static final String JSON_CONFIG_FILE = "config.json";
+
+    @Override
+    protected void waitUntilPageObjectIsLoaded() {
+        webDriverWait.until(ExpectedConditions.elementToBeClickable(saveButton));
+    }
+
+    /**
+     * Set a text box.
```
Comment (JuanCarlosMartinez4) [inline src/main/java/salesforce/ui/pages/owner/OwnerEditClassicPage.java]: improve this comment please

File: src/test/java/salesforcetest/steps/OpportunityStep.java [hunk 2]
```diff
@@ -81,11 +84,13 @@ public void changesAnOpportunitySOwnerWith(final String ownerType) {
...
-        String expectedResult = ReplacerMessages.replaceChangeOwnerMessage(message, context.getOpportunities()
-                .get(FIRST_OPPORTUNITY).getName());
-        Assert.assertEquals(actualResult, expectedResult);
+        if (userExperience.equals(USER_EXPERIENCE_LIGHTNING)) {
+            ToastMessageSpan toastMessageSpan = new ToastMessageSpan();
+            String actualResult = toastMessageSpan.getToastMessage();
+            String expectedResult = ReplacerMessages.replaceChangeOwnerMessage(message, context.getOpportunities()
+                    .get(FIRST_OPPORTUNITY).getName());
+            Assert.assertEquals(actualResult, expectedResult);
+        }
```
Comment (JuanCarlosMartinez4) [inline src/test/java/salesforcetest/steps/OpportunityStep.java:92]: if the user experience is not Lightning, what happened?

Comment (Enrique-C) [reply src/test/java/salesforcetest/steps/OpportunityStep.java:92]: Classic user experience don't have a successful message

Comment (JuanCarlosMartinez4) [reply src/test/java/salesforcetest/steps/OpportunityStep.java:92]: But you need to validate something right?

Comment (Enrique-C) [reply src/test/java/salesforcetest/steps/OpportunityStep.java:92]: I have 2 validation, but 1 only for classic, and 2 for lightninh

Comment (JuanCarlosMartinez4) [reply src/test/java/salesforcetest/steps/OpportunityStep.java:92]: where is the classic UE validation?

File: src/test/resources/features/Opportunity.feature [hunk 3]
```diff
@@ -50,6 +50,10 @@ Feature: Opportunity
     Then On the details section should display the Campaign name
 
 
+  Scenario: Add new Task with required values
+    Given
+
+
```
Comment (JuanCarlosMartinez4) [inline src/test/resources/features/Opportunity.feature]: delete this scenario if you are not using it.

**Review by JuanCarlosMartinez4 — APPROVED** [review]

**Pull request merged**